internal/server/get_downloads.go

```diff
@@ -1,10 +1,16 @@
 package server
 
 import (
+	"github.com/opg-sirius-finance-admin/internal/model"
 	"net/http"
 )
 
 type GetDownloadsVars struct {
+	ReportsTypes        *[]model.ReportsType
```

Comment:
a slice is already a pointer, so I would change these to `[]` instead of `*[]`

Reply:
done